porch/api/porch/v1alpha1/types_packagerevisions.go

```diff
@@ -41,14 +41,22 @@ type PackageRevisionList struct {
 	Items []PackageRevision `json:"items"`
 }
 
-// PackageRevisionSpec defines the desired state of PackageRevision
-type PackageRevisionSpec struct {
-	PackageName string `json:"packageName,omitempty"`
+type PackageRevisionType string
 
-	Revision string `json:"revision,omitempty"`
+const (
+	PackageRevisionTypeDraft    PackageRevisionType = "draft"
```

Comment:
I _think_ we generally capitalize user-visible "enums" (e.g. Draft, Proposed, Final)

Reply:
Looking at prior art, this is correct. I fixed these constants but there are more to fix too that we currently lower-case. I'll add tracking issue for that.